### PR TITLE
Estelle/webkit/1444139

### PR DESCRIPTION
**Pull request by estelle**
added the re-enabling of layout.css.prefixes.device-pixel-ratio-webkit (i.e. add back support for -webkit-min-device-pixel-ratio) in FF 63, with no flag.

### CODE REVIEW

File: api/MediaCapabilities.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
+{
```
Comment (ddbeck) [inline api/MediaCapabilities.json]: Was this file erroneously included in the PR? It doesn't seem relevant to https://bugzilla.mozilla.org/show_bug.cgi?id=1444139

File: css/at-rules/media.json [hunk 2]
```diff
@@ -1605,26 +1605,11 @@
...
-                    {
-                      "type": "preference",
-                      "name": "layout.css.prefixes.device-pixel-ratio-webkit",
-                      "value_to_set": "true"
-                    }
-                  ],
-                  "notes": [
-                    "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
-                  ]
+                  "version_added": "63"
```
Comment (Elchi3) [inline css/at-rules/media.json]: If I understand it correctly, here's how this was implemented:

- Implemented by default in Firefox 63+ _(this is new)_.
- Implemented behind one flag (layout.css.prefixes.device-pixel-ratio-webkit) in Firefox 49+.
- Implemented behind two flags (layout.css.prefixes.device-pixel-ratio-webkit and layout.css.prefixes.webkit) in Firefox 45+.

I would represent these support ranges like this:

```diff
"firefox": [
+  {  
+    "version_added": "63",
+    "notes": [
+      "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
+    ]
+  },
  {
    "version_added": "49",
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.prefixes.device-pixel-ratio-webkit",
        "value_to_set": "true"
      }
    ],
    "notes": [
      "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
    ]
  },
  {
    "version_added": "45",
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.prefixes.webkit",
        "value_to_set": "true"
      },
      {
        "type": "preference",
        "name": "layout.css.prefixes.device-pixel-ratio-webkit",
        "value_to_set": "true"
      }
    ],
    "notes": [
      "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
    ]
  }
],
```

File: css/at-rules/media.json [hunk 3]
```diff
@@ -1605,26 +1605,11 @@
               },
               "firefox": [
                 {
-                  "version_added": "49",
```
Comment (ddbeck) [inline css/at-rules/media.json]: Can you help me understand the data that's dropped here (i.e., commit 4468d5f261f995759c00af19d4339149d45576a5)? The existing data seems approximately right (i.e., you used to have to enable a pref to get this feature)? Was there something wrong with this data after all?

**Comment by Elchi3** [comment]
Whoops review collision! :-) Cheers Daniel! :rofl:

**Comment by ddbeck** [comment]
@Elchi3's review was a bit more incisive. Feel free to ignore my questions, @estelle 😄

**Comment by estelle** [comment]
Thank you for the feedback.

I opened up PR ##2581 because it was faster than figuring out how to remove the media capabilities PR from this one.
